redbot/cogs/mutes/mutes.py

```diff
@@ -0,0 +1,1219 @@
+import asyncio
+import contextlib
+import discord
+import logging
+
+from abc import ABC
+from copy import copy
+from typing import cast, Optional, Dict, List, Tuple, Literal, Coroutine, Union
+from datetime import datetime, timedelta, timezone
+
+from .converters import MuteTime
+from .voicemutes import VoiceMutes
+
+from redbot.core.bot import Red
+from redbot.core import commands, checks, i18n, modlog, Config
+from redbot.core.utils.chat_formatting import humanize_timedelta, humanize_list, pagify
+from redbot.core.utils.mod import get_audit_reason
+from redbot.core.utils.menus import start_adding_reactions
+from redbot.core.utils.predicates import MessagePredicate, ReactionPredicate
+
+T_ = i18n.Translator("Mutes", __file__)
+
+_ = lambda s: s
+mute_unmute_issues = {
```

Comment:
Since this is a global constant, it should be all uppercase.
```suggestion
MUTE_UNMUTE_ISSUES = {
```